backend/dataall/modules/datasets/api/table/enums.py

```diff
@@ -1,12 +1,6 @@
 from dataall.base.api.constants import GraphQLEnumMapper
 
 
-class DatasetSortField(GraphQLEnumMapper):
-    created = 'created'
-    updated = 'updated'
-    label = 'label'
-
-
 class DatasetTablePreviewStatus(GraphQLEnumMapper):
```

Comment:
maybe move this enum to `datasets_base/constants/enums.py ` and get rid of this file?